### DISCUSSION TITLE
迅雷云盘无法使用，“有偿”求搭理，急

### DISCUSSION THREAD

**Discussion by snow212-cn** [AlistGo/alist#795] [Ideas]
填了迅雷账号密码，卡在了登录
![image](https://user-images.githubusercontent.com/18443918/160057200-6e81766e-3fc6-4185-b6b1-d4c925dd4bdb.png)
浏览器打开无法显示文件
![image](https://user-images.githubusercontent.com/18443918/160057118-37a3ed19-ce17-4cfa-b274-b3f13e432832.png)

**Comment by raygzcn** [comment]
同样是迅雷遇到问题
